### PR TITLE
update operator readme.md

### PR DESCRIPTION
**Pull request by wlp1153468871**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Refer to https://github.com/karmada-io/karmada/pull/3577#issuecomment-1605893356.

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by RainbowMango — COMMENTED** [review]
/assign @calvin0327

File: operator/README.md [hunk 1]
```diff
@@ -136,6 +136,13 @@ It is important to proceed with caution when deleting a Karmada CR due to the po
 kubectl delete karmada karmada-demo -n test
 ```
 
+If you want to delete a Karmada CR without cascading deletion of the associated Karmada instances,
```
Comment (calvin0327) [inline operator/README.md:139]: `instances` ->  `instance`

I think it should be singular here.

**Comment by calvin0327** [comment]
@windsonsea @Poor12 /cc

File: operator/README.md (hunk 1 above)
Comment (wlp1153468871) [reply operator/README.md]: thanks,done

File: operator/README.md [hunk 2]
```diff
@@ -136,6 +136,13 @@ It is important to proceed with caution when deleting a Karmada CR due to the po
 kubectl delete karmada karmada-demo -n test
 ```
 
+If you want to delete a Karmada CR without cascading deletion of the associated Karmada instance,
+you can execute the following command before performing the deletion operation.
```
Comment (windsonsea) [inline operator/README.md:140]:
```suggestion
you can run the following command before performing the deletion operation.
```
Shorter is better in addition to style consistency:

image

**Review by windsonsea — COMMENTED** [review]
Pick a nit.

I search through our repo, few `cascading deletion` is found. Probably, we can find somewhere to explain what is cascading deletion. ChatGPT tells me:

Cascading deletion is a feature in Karmada that allows deleting child resources along with their parent resources automatically. When a user deletes a parent resource, all the associated child resources are also automatically deleted without any manual intervention. 

For example, when a user deletes a namespace in Karmada, all the deployments, services, and other resources within that namespace will be deleted automatically. This helps to ensure clean-up of resources and avoid orphaned resources in the system.

Cascading deletion can be enabled or disabled for each individual resource type in Karmada YAML file by configuring the `cascade` field. By default, cascading deletion is enabled for most resource types.

File: operator/README.md (hunk 2 above)
Comment (wlp1153468871) [reply operator/README.md]: thanks,done

**Comment by windsonsea** [comment]
/lgtm
for the text excluding command

**Comment by calvin0327** [comment]
@wlp1153468871 please retrigger the ci workflow.

**Comment by RainbowMango** [comment]
Echo the logs before retrigging CI:
```
Summarizing 2 Failures:
  [INTERRUPTED] [ExplicitPriority] propagation testing high explicit/low priority/implicit priority PropagationPolicy propagation testing [It] high explicit/low priority/implicit priority PropagationPolicy propagation testing
  /home/runner/work/karmada/karmada/test/e2e/propagationpolicy_test.go:771
  [FAIL] [ClusterAffinities] propagation testing when [PropagationPolicy] replica scheduling type is Duplicated schedule with multi clusterAffinity [It] propagate deployment and then update the cluster label
  /home/runner/work/karmada/karmada/test/e2e/framework/deployment.go:108

Ran 66 of 139 Specs in 646.196 seconds
FAIL! - Interrupted by Other Ginkgo Process -- 64 Passed | 2 Failed | 0 Pending | 73 Skipped
```

This flaking test is tracked by https://github.com/karmada-io/karmada/pull/3518.

**Review by RainbowMango — APPROVED** [review]
/approve
Seems no other comments left.
Thanks.

**Pull request merged**